client/src/App.jsx

```diff
@@ -1,14 +1,21 @@
 import { Header } from "./components/Header/Header.jsx";
+import { LandingPage } from "./components/LandingPage/LandingPage.jsx";
 import { ImageGenerator } from "./components/ImageGenerator/ImageGenerator.jsx";
```

Comment:
None of these need the `jsx` extension

```suggestion
import { Header } from "./components/Header/Header";
import { LandingPage } from "./components/LandingPage/LandingPage";
import { ImageGenerator } from "./components/ImageGenerator/ImageGenerator";
```